caddy/https/setup.go

```diff
@@ -220,6 +228,11 @@ func loadCertsInDir(c *setup.Controller, dir string) error {
 // port to 443 if not already set, TLS is enabled, TLS is manual, and the host
 // does not equal localhost.
 func setDefaultTLSParams(c *server.Config) {
+	// If no KeyType provided, use EC384 as preferred KeyType
+	if KeyType == "" {
+		KeyType = acme.EC384
+	}
```

Comment:
How is the compatibility for EC certs? Maybe we should still default to 2048 bit RSA. What do you think?